src/scripts/hubot-jenkins-notifier.coffee

```diff
@@ -115,24 +144,51 @@ class JenkinsNotifier
     [@dataMethodJSONParse, @dataMethodRaw].forEach(filterChecker)
 
     if !data || typeof data.build != 'object'
-      @error new Error("Unable to process data"), req.body
+      @error new Error("Unable to process data - data empty or not an object"), req.body
       return
+
+    @logMessage(query.trace, data.build)
+
     fullurl = data.build.full_url || data.build.url
 
+    if data.build.phase in ['COMPLETED']
+      @logMessage(query.trace, "Ignoring phase COMPLETED")
+      return
+
+    @logMessage(query.trace, "#{data.name} #{data.build.phase} #{data.build.status}")
+
+    if data.build.phase in ['STARTED']
+      @robot.send envelope, "#{data.name} build ##{data.build.number} started: #{fullurl}"
```

Comment:
;) Yes you did.
I wasn't so sure about this, since shouldNotify respects the current and the previous build outcome. But STARTED does not have an outcome, right?